### DISCUSSION TITLE
Type /skyblock when entering the server

### DISCUSSION THREAD

**Discussion by RamazanQs** [ORelio/Minecraft-Console-Client#2083] [Q&A] [answered]
How can I automatically print /skyblock when logging into the server

**Comment by BruceChenQAQ** [comment]
~~If repeated execution of this command has no side effects, you can use AntiAFK~~

You can use Script Scheduler / Task Scheduler

doc: https://github.com/MCCTeam/Minecraft-Console-Client/tree/master/MinecraftClient/config#using-the-script-scheduler--task-scheduler

https://github.com/MCCTeam/Minecraft-Console-Client/blob/master/MinecraftClient/config/sample-tasks.ini

**Comment by milutinke** [comment, marked as answer]
Do this: https://github.com/MCCTeam/Minecraft-Console-Client/discussions/2091#discussioncomment-3392546
But instead of:
```
wait 30
send /login yourpasswordhere
```

Use:
```
wait 1
send /skyblock
```

**Reply by softwaredeveloper222** [reply]
wait 2
send /register
